src/utils.sh

```diff
@@ -440,3 +453,41 @@ function handlers_init() {
 }
 
 export -f handlers_init
+
+# Constants
+InstrumentationKey="e83c2a5c-35d1-4a37-a5fe-b4f0f6e8e2a5"
+IngestionEndpoint="https://dc.services.visualstudio.com/v2/track"
+EventName="Azure-IoT-Edge-Installer-Summary"
+DeviceUniqueID="xinzedPC"
+CurrentTime=$(echo `date '+%Y-%m-%dT%H:%M:%S.%N'`)
```

Comment:
>echo `date '+%Y-%m-%dT%H:%M:%S.%N'` [](http://example.com/codeflow?start=14&length=35)

I think we should log in utc - $(date --utc +%FT%T.%3NZ)

Reply:
addressed in new commit